tests/unit/linalg/test_matrix_dense.cpp

```diff
@@ -245,3 +246,51 @@ TEST_CASE("LUFactors RightSolve", "[DenseMatrix]")
 
    REQUIRE(C.MaxMaxNorm() < tol);
 }
+
+TEST_CASE("DenseTensor LinearSolve methods",
```

Comment:
It might be nice in the future to have test cases that cover capabilities meant to be run on the GPU to be run using the debug device option. I didn't see anything here that might cause issues, but it's just something that would be nice to catch any problems or regressions.

Reply:
Could that be done using travis and a debug build?

Reply:
I don't see why it couldn't. Although, we'll probably want to look into this issue with a separate PR. Since, I'm sure we'll turn up a number of potential bugs that need to be fixed.